tests/40-jcs-create.js

```diff
@@ -0,0 +1,177 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
```

Comment:
again, `copyright W3C CC-whatever` also/instead

Reply:
Added `LICENSE.md` and updated the copyright headers here - [2f768fa](https://github.com/w3c/vc-di-eddsa-test-suite/pull/32/commits/2f768fabbb79539cad13543662844166785420fb)